### PR TITLE
Add iban and docs

### PR DESCRIPTION
**Pull request by martijn00**
## Connection with issue(s)

Close #???

Connected to #???

## To Do

- [ ] Read [contributing guide](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/CONTRIBUTING.md)
- [ ] Check the original issue to confirm it is fully satisfied
- [ ] Add solution description to help guide reviewers
- [ ] Add unit test to verify new or fixed behaviour
- [ ] If apply, add documentation to code properties and package readme

### CODE REVIEW

File: lib/src/utils/validators.dart [hunk 1]
```diff
@@ -427,3 +427,31 @@ bool isMACAddress(String value) {
...
+  }
+
+  String rearranged = iban.substring(4) + iban.substring(0, 4);
+  String numericIban = rearranged.split('').map((char) {
+    int charCode = char.codeUnitAt(0);
+    if (charCode >= 65 && charCode <= 90) {
+      return (charCode - 55).toString(); // A=10, B=11, ..., Z=35
+    } else {
+      return char;
+    }
```
Comment (deandreamatias) [inline lib/src/utils/validators.dart:445]: Can be a ternary if else

File: test/form_builder_validators_test.dart [hunk 2]
```diff
@@ -1001,4 +1001,111 @@ void main() {
...
+      final validator = FormBuilderValidators.log<String>(
+        log: (value) => 'Logging: $value',
+      );
+      // Pass
+      expect(validator('test'), isNull);
+      // Fail
+      expect(validator(null), isNull); // Log message will be displayed
+    }),
+  );
 }
```
Comment (deandreamatias) [inline test/form_builder_validators_test.dart:1111]: I missed the test of IBAN validator

**Review by deandreamatias — COMMENTED** [review]
I remember about a uppercase and lowercase validation to take in account on uppercaseCharLength and lowercaseCharLength regex. Maybe on _alphabetical regex too.

https://github.com/mrmilu/flutter_mrmilu/blob/03fa62cb531b52237b9d035c48654b4ac286136e/lib/src/validators/password_validators.dart#L4

**Comment by martijn00** [comment]
@deandreamatias Nice! Can you make the new release? Can't wait to try it out!

**Pull request merged**